components/MetroBarChart.vue

```diff
@@ -2,15 +2,52 @@
   <data-view :title="title" :title-id="titleId" :date="date">
     <template v-slot:button>
       <p class="MetroGraph-Desc">
-        {{ chartData.base_period }}の利用者数*の平均値を100としたときの相対値
+        {{
+          $t('{range}の利用者数*の平均値を基準としたときの相対値', {
+            range: chartData.base_period
```

Comment:
シートのRow 45に「1月20日\~1月24日」の翻訳があるので、「1月20日\~1月24日」も辞書に登録してもらって
```suggestion
            range: $t(chartData.base_period)
```

みたいな感じで`chartData.base_period` も翻訳しちゃってください（応急措置）
良いやり方ではないですが... `data/*.json`周りのデータ構造が変わったら直すということで

Reply:
@MaySoMusician 日付の設定も追加しました